index.bs

```diff
@@ -355,6 +371,53 @@ or failure:
 	append a duplicate of the second element to |tokens|.
 7. Return |tokens|.
 
+The <dfn for="IntersectionObserver">scrollable intersection rectangles</dfn>
+for an {{IntersectionObserver}}
+are the rectangles of scrollable containers on the path from the
+<a>intersection root</a> to <a for="IntersectionObserver">target</a>
+which could clip the target's rectangle.
+
+When calculating the <a>scrollable intersection rectangles</a> for
+a <a>same-origin-domain target</a>, the rectangle is then expanded
+according to the offsets in the {{IntersectionObserver}}’s {{[[scrollMargin]]}} slot
+in a manner similar to CSS's 'margin' property,
```

Comment:
Link to:

https://www.w3.org/TR/css-box-3/#margins

Reply:
Done